src/components/ActiveProject/ActiveProject.js

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
This file should be a jsx file. I also think it might have too many spaces on each line, so we should check that too!